getodk/collect
# Fix: Popup menu in Import/Export Settings exists only for one item #1857

Closes #1857 

#### What has been done to verify that this works as intended?
Steps to be followed
1. Go to Admin Settings -> Import/Export Settings
2. Open the popup menu.
Here you can check the icons in the top bar. Below is a comparison of screens before and after the fix
Before->
![wadasdsa](https://user-images.githubusercontent.com/10618953/36203693-74c2eb42-11ae-11e8-8c70-e6194cd91542.png)
After->
![screen_dark](https://user-images.githubusercontent.com/10618953/36203814-e87c83cc-11ae-11e8-8ea8-d438048235e3.png)

#### Why is this the best possible solution? Were any other approaches considered?
This is the best solution as its using vector graphics instead of PNGs. Vector graphics are more flexible and take less memory. They are automatically resized according to the screen size and pixel density. 
Another approach is to use PNGs but then we have to create different icons to support all screen sizes. 
 
#### Are there any risks to merging this code? If so, what are they?
No, there are no risks in merging my code as such.
But please check some of the changes that i have done to from my side to the code to make it less redundant and clean.
- Replaced ic_menu_save icon with the vector in form menu.
- Placed Google api key in Strings,xml to create a level of abstraction. 

#### Do we need any specific form for testing your changes? If so, please attach one.
No, as there are no forms involved with this screen.
